assets/src/dashboard/components/pill/components.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as CheckmarkIcon } from '../../icons/checkmark.svg';
+import { ReactComponent as CloseIcon } from '../../icons/close.svg';
+import { BEZIER } from '../../constants';
+
+export const ACTIVE_CHOICE_ICON_SIZE = 16;
+export const ACTIVE_CHOICE_LEFT_MARGIN = 4;
+
+export const PillContainer = styled.label`
+  ${({ theme }) => `
+    display: inline-flex;
+    justify-content: center;
+    font-family: ${theme.fonts.pill.family};
+    font-weight: ${theme.fonts.pill.weight};
+    font-size: ${theme.fonts.pill.size}px;
+    line-height: ${theme.fonts.pill.lineHeight}px;
+    letter-spacing: ${theme.fonts.pill.letterSpacing}em;
+`}
+`;
+
+export const PillInput = styled.input`
+  /*
+Hide checkbox visually but remain accessible to screen readers.
+Source: https://polished.js.org/docs/#hidevisually
+*/
+  border: 0;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`;
+
+export const PillLabel = styled.span`
+  ${({ theme, isSelected }) => `
+  cursor: pointer;
+  margin: auto 0;
+  width: 100%;
+  display: flex;
+  padding: 4px;
+  padding-right: ${isSelected ? '4px' : '20px'};
```

Comment:
Unless the style requires computed values from the props we destructured let’s make it top level in the parent ttl. The inner styles will get recalculated when the props change. It creates a static style for the selectors that are static and inline ones for the variable ones. Also here since the entire parent tagged template literal just wraps a function you could do 
```
styled.div(({ props }) => ` ... ` );
```